CMakeLists.txt

```diff
@@ -54,6 +54,15 @@ message(STATUS "BUILD_TESTING: ${BUILD_TESTING}")
 option(BUILD_TESTING_INTEGRATION "" OFF)
 message(STATUS "BUILD_TESTING_INTEGRATION: ${BUILD_TESTING_INTEGRATION}")
 
+# build header only cpp bindings
+option(BUILD_CPP_BINDINGS "" OFF)
+if (BUILD_TESTING OR BUILD_TESTING_INTEGRATION)
+  set(BUILD_CPP_BINDINGS ON)
+  message(STATUS "Testing triggering cpp binding dependancy.")
```

Comment:
Do we need the message?

Reply:
Probably not, but it seemed convenient to inform the user about it.